src/components/inputs/checkbox.vue

```diff
@@ -0,0 +1,28 @@
+
+<template>
+    <div :class="$attrs.class">
```

Comment:
move it to the input `:class="$attrs.class"` and keep a static class on div class="km-controls km-checkbox"
we need a good name for common css selectors class(we can decide that later)